flight/Libraries/math/pid.c

```diff
@@ -199,6 +211,39 @@ void pid_configure(struct pid *pid, float p, float i, float d, float iLim)
 }
 
 /**
+ * Set the deadband values
+ * @param[out] pid The PID structure to configure
+ * @param[in] deadband Deadband width in degrees per second
+ * @param[in] slope Deadband slope (0..1)
+ */
+void pid_configure_deadband(struct pid_deadband *deadband, float width, float slope)
+{
+#ifndef SMALLF1
```

Comment:
Likewise-- I don't think this should be here.  If we want to prohibit pid deadbands on F1, the caller should not do it, so we can do it in cases that make sense.

But I think our discussion and a lot of the memory optimization has been with the goal of running this on F1.